src/manager.js

```diff
@@ -412,14 +451,19 @@ AdManager.prototype.loadAds = function(element, updateCorrelator) {
 
     var slot = this.configureAd(thisEl);
 
-    if (typeof window.index_headertag_lightspeed === 'undefined') {
+    if (slot) {
+      slotsToLoad.push(slot);
+    }
+
+    if (typeof window.headertag === 'undefined' || window.headertag.apiReady !== true) {
       this.googletag.display(thisEl.id);
     } else {
-      window.index_headertag_lightspeed.slotDisplay(thisEl.id, ads);
+      window.headertag.display(thisEl.id);
```

Comment:
@spra85 This will definitely work on kinja but i'm unsure about the bulbs sites.  I'll ask during the weekly meeting but in the past when i've asked them their team didn't acknowledge that there were any differences between implementations.  I think the safest thing to do would be to test this ourselves on the onion sites to make sure things work as expected. I can help with this if need be.